### DISCUSSION TITLE
Proposal: Map Sync/Exporting list

### DISCUSSION THREAD

**Discussion by MoonLGH** [ppy/osu#21155] [Ideas] [answered]
continuing #2351
I think it's a good idea to create make these features, personally, I'm the type of person who changes OS every 1-2 months once, and sometimes I forget to copy the song folder, I actually also make a third-party app to fix this problem, by extracting each one Beatmap from stable/songs to get its id and extract it to JSON, and just download it by its JSON each one if wanted to
My examples [BeatmapDL](https://github.com/MoonLGH/BeatmapDownloader) on src/index, src/main/lists
I would love to work on these on Lazer if I had time to

**Comment by MoonLGH** [comment]
also, the examples i put on top only can be used on stable

**Comment by peppy** [comment, marked as answer]
You can keep a copy of your `client.realm`, or backup your data folder.

**Reply by MoonLGH** [reply]
where is `client.realm` ... my lazer folder looks like this
![image](https://user-images.githubusercontent.com/66781351/200323599-131772ca-4ead-44de-959e-f67de7d41a5b.png)
is it the one in roaming ? if yes then... there is nothing on that folder/file

**Reply by MoonLGH** [reply]
after looking at my storage.ini everything make sense now, okay

**Reply by Axilotl17** [reply]
> You can keep a copy of your `client.realm`, or backup your data folder.

doesnt that only save a list, and then you have to download every one again? when i did that i saw what maps i had but they all failed to load, as all the data was in the hashed folder

oooh, can you just update them all? I ran into this issue before that was implimented

**Reply by peppy** [reply]
Eventually the game will support that yes. If any files are missing you will have the option to repair the beatmap.
